actors/badge/src/test/java/org/sunbird/badge/actors/BadgeAssertionActorTest.java

```diff
@@ -0,0 +1,137 @@
+package org.sunbird.badge.actors;
```

Comment:
Need to add some test case for invalid user id , invalid badgeId,  assertion for valid contentId, assertion for invalid contentId.